Add README (#207)
home-assistant/yellow.home-assistant.io: README.md

# Home Assistant Yellow website

This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 

## Setup

This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.

> ## Development

Reviewer: Might it be an idea to by default point to codespaces?

Especially if one would just make a single edit quickly, you don't need anything and just start typing working directly from your browser...

# Home Assistant Yellow website

This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 

## Setup

This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.

## Development

> The easiest way to edit the documentation is to open the environment in Github Codespaces. Codespaces work in your browser, no installation required.

Reviewer 2: ```suggestion
The easiest way to edit the documentation is to open the environment in GitHub Codespaces. Codespaces work in your browser, no installation required.
```

1. Further up on this page, select the green **Code** button. 

> 2. On the **Codespaces** tab, select **Open in codesapce**.

Reviewer 2: ```suggestion
2. On the **Codespaces** tab, select **Create codespace on main**.
```

Reviewer 2: Presumably I'm getting this button text since I've never used Codespaces for this repo before

3. Make your changes and push them.
4. You are amazing! Thank you for contributing.

> 

Reviewer 2: ```suggestion
```

If you prefer to run this locally, the recommended development environment is Visual Studio Code. The repository uses [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers). When opening a checked out version of this repository, make sure to open it in the Dev Container (Ctrl+Shift+P > **Dev Containers: Open Folder in Container...**)

> If you prefer other editors, Dev Containers is an open standard and can be used with other editors. See [containers.dev](ttps://containers.dev/) for more information on supported editors.

Reviewer 2: ```suggestion
If you prefer other editors, Dev Containers is an open standard and can be used with other editors. See [containers.dev](https://containers.dev/) for more information on supported editors.
```
